logging/src/main/java/org/jboss/as/logging/LogFileResourceDefinition.java

```diff
@@ -257,7 +258,7 @@ public void execute(final OperationContext context, final ModelNode operation) t
             }
             try (
                     final InputStream in = (tail ? new LifoFileInputStream(file) : Files.newInputStream(file.toPath()));
-                    final InputStreamReader isr = (encoding == null ? new InputStreamReader(in) : new InputStreamReader(in, encoding));
+                    final InputStreamReader isr = (encoding == null ? new InputStreamReader(in, StandardCharsets.UTF_8) : new InputStreamReader(in, encoding));
```

Comment:
I think we should stick with the default here and not use UTF-8. The encoding on the file handler does not default to UTF-8 but the system default. I think here we should stick with the system default unless explicitly defined. I could see a UTF-8 default possibly being problematic on IBM bases systems.